src/containers/MyProjectsPage.js

```diff
@@ -0,0 +1,37 @@
+import React, { Component } from 'react'
```

Comment:
__[eslint]__
* Error - Unable to resolve path to module 'react'. (import/no-unresolved)

<!-- Comment by [SideCI](https://sideci.com) -->